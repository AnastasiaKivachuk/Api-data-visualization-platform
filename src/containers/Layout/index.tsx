import type {NextPage} from 'next'
import React from "react";
import styled from "@emotion/styled";

import Grid from '@mui/material/Grid'
import {Container} from "@mui/material";

import Menu from "../../components/Menu";

type Props = {
    children: JSX.Element
}

const Layout: NextPage<Props> = ({children}) => {
    const StyledContainer = styled(Container)`
      padding-top: 60px;
      display: flex;
      flex-direction: column;
      align-items: center;
    `
    return (
        <Grid item>
            <Menu/>
            <StyledContainer>{children}</StyledContainer>
        </Grid>
    )
}

export default Layout
