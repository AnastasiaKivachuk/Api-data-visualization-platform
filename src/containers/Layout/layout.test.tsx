import {shallow} from "enzyme";
import React from "react";

import Layout from "./index";

const component = shallow(<Layout ><div>test Component</div></Layout>);

describe("Layout component", () => {

    it("should render Layout component", () => {
        expect(component).toMatchSnapshot();
    })
})
