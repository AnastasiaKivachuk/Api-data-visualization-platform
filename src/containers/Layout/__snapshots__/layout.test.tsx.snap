// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should render Layout component 1`] = `
<ForwardRef(Grid)
  item={true}
>
  <Menu />
  <Styled(Component)>
    <div>
      test Component
    </div>
  </Styled(Component)>
</ForwardRef(Grid)>
`;
